data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+			`	As it is currently lunch time, there is no one in sight at the clinic. Luckily, the operator gave you the room number, which you quickly reach after a few minutes.`
+			choice
+				`	(Wait for someone to show up.)`
+					goto wait
+				`	(Enter the room.)`
+			apply
+				set "james medicine: infected"
+			`	You enter the room and find that James is inside, lightly asleep. He wakes up just as you close the door.`
+			`	"<first>! Fancy meeting you here! I heard you became a war hero of sorts?", he adds with a wink.`
+			choice
+				`	"Nothing special, I just did what is right."`
+				`	"Indeed, seems like I saved the galaxy this time."`
+			`	"Yeah, yeah... You know, I am feeling pretty weak now, and while the doctors say that I will live through this illness, there is still a chance that it would be my last."`
+			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but.. there is a wormhole there!"`
+			branch interesting
+				not "First Contact: Hai: offered"
+			choice
+				`	(Tell him about the Hai.)`
+				`	(Say nothing about it.)`
+					goto interesting
+			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
+			`	"Indeed? Are they friendly?"`
+			choice
+				`	"Yes, they even offer humans permission to settle there or buy their technology."`
+					goto doctor
+				`	"Yes, though I'm worried they can conquer all of human space if they wish."`
+					goto doctor
+			label interesting
+			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be settled?"`
+			`	"Not sure. It is located in a lawless barren region of space, however, so I wouldn't be surprised if there pirates or even Alphas on the other side."`
+			label doctor
+			`	Just as the conversation hangs for a moment, the door opens and a doctor in protective clothing steps in.`
+			`	"Who are you? And why are you in this room?", the doctor inquires, in a soft but authoritative voice.`
+			choice
+				`	"I am <first> <last>, here to see this friend of mine."`
+				`	"I was.. just strolling around."`
+			`	"Nevermind what you were doing. I have to warn you that the patient here has a disease we do not know much about yet. It could potentially be some deadly, contagious disease. Now, please make haste and remove yourself from the room, so as to not risk catching it yourself."`
+			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you hear something of a chuckle, probably from James.`
```

Comment:
If we're dealing with this kinda doctor, why not let him shove us out the room while we still try to process the news?